net/acme/files/run.sh

```diff
@@ -76,6 +81,17 @@ pre_checks()
                 return 1
             fi
             ;;
+		"nginx")
+			debug "Found nginx listening on port 80; trying to disable."
+			local tries=0
+			while [ "$(pgrep nginx)" ];
+				tries=$tries+1
+				/etc/init.d/nginx stop
+				if [ $tries -gt 5 ]; then
+					debug "Can't stop nginx. Terminating script."
+					return 1
+				fi
+			done
```

Comment:
Erm, why the loop and multiple executions of the stop script?

Reply:
nginx is slow to close... i can put a sleep instead of another stop. Better?